src/wallet/wallet.h

```diff
@@ -657,6 +657,8 @@ class CWallet final : public CCryptoKeyStore, private interfaces::Chain::Notific
     int64_t nNextResend = 0;
     int64_t nLastResend = 0;
     bool fBroadcastTransactions = false;
+    // Local time that the tip block was received. Used to schedule wallet rebroadcasts.
+    int64_t m_best_block_time {0};
```

Comment:
This is accessed from two different threads so it might be better if this were atomic or accessed with the cs_wallet lock.

Reply:
Good catch! I'll make this atomic.